packages/components/app/styles/components/tabs.scss

```diff
@@ -66,10 +66,10 @@
 
 .hds-tabs__tab-button {
   @include hds-focus-ring-with-pseudo-element(
-    $top: var(--token-tabs-tab-focus-inset),
-    $right: var(--token-tabs-tab-focus-inset),
-    $bottom: var(--token-tabs-tab-focus-inset),
-    $left: var(--token-tabs-tab-focus-inset),
+    $top: -2px,
+    $right: -6px,
```

Comment:
I'm not entirely sure why these values need adjustments tbh, but these are the values that result in no visual regression for Tabs.

These two values can be converted to tokens (e.g. `--token-tabs-tab-focus-inset-horizontal`, `--token-tabs-tab-focus-inset-vertical`) and `--token-tabs-tab-focus-inset` can be removed.

Reply:
Do you have this tracked in a JIRA issue?

Reply:
Planning to address it in this PR, but wanted to check with the team first.